src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputLineLengthIgnoringPackageStatements.java

```diff
@@ -0,0 +1,26 @@
+/*
+LineLength
+fileExtensions = (default)all files
+ignorePattern = (package|//).*
+max = 58
+
+
+*/
+          package com.puppycrawl.tools.checkstyle.checks.sizes.linelength; // ok
+
+// violation below 'longer than 58 characters (found 59)'
+import java.security.interfaces.RSAMultiPrimePrivateCrtKey;
+
+import java.util.Arrays; // ok
+
+// violation below 'longer than 58 characters (found 77)'
+                  import java.security.interfaces.RSAMultiPrimePrivateCrtKey;
+
+public class InputLineLengthIgnoringPackageStatements {
+    @Override
+    public String toString() {
+
+        // violation below 'longer than 58 characters (found 77)'
+        return "This line is longer than 60 characters and should be logged";
```

Comment:
make code where import/package is used as word of variable and `String s  = " import ";`, to show that we do not higlight them.

and no violation on:
```
String s2 = """
    import long long long text
""";
```

violation on:
```
String s2 = """ text
import long long long text without indentation
     indented text""";
```

Reply:
Done